ChatApp/ChatApp/Credentials.swift.example

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright IBM Corporation 2016
+ * Copyright IBM Corporation 2017
```

Comment:
I think we probably want the copyright date to be "2016,2017".

Reply:
Just pushed another commit to update the copyright date. :+1: